Framework/Beamline/inc/MantidBeamline/ComponentInfo.h

```diff
@@ -112,6 +114,7 @@ class MANTID_BEAMLINE_DLL ComponentInfo {
                    const Eigen::Quaterniond &newRotation);
 
   size_t parent(const size_t componentIndex) const;
+  bool isVisible(const size_t componentIndex) const;
```

Comment:
I'm not sure about this as `isVisible`. This pertains only to the `InstrumentView`. I think leave in place for now though.